subt/main.py

```diff
@@ -293,6 +311,14 @@ def update(self):
                 self.scan = data
                 if self.local_planner is not None:
                     self.local_planner.update(data)
+            elif channel == 'scan360':
+                if self.flipped:
+                    mid = len(data)//2
+                    self.scan = (data[mid:]+data[:mid])[30:-30]
+                else:
+                    self.scan = data[30:-30]
```

Comment:
What is the intent here? I understand `data` contains scan of 360°. If the intent is to convert it to 270°, the value `30` should depend on `len(data)`.

Reply:
Good point - this was hard-coded for Robik, which is currently the only robot having 360 FOV lidar. And yes, the idea is to get "scan as usual", which is 270 degrees for all SICK lidars including virtual world.